front/app/containers/Admin/projects/project/participationContext/components/ParticipationMethodPicker.tsx

```diff
@@ -140,23 +140,25 @@ export const ParticipationMethodPicker = ({
               }
             />
           </FeatureFlag>
-    {showSurveys && (
-      <ParticipationMethodRadio
-        onChange={handleParticipationMethodOnChange}
-        currentValue={participation_method}
-        value="survey"
-        name="participationmethod"
-        id={'participationmethod-survey'}
-        label={
-          <LabelHeaderDescription
-            header={<FormattedMessage {...messages.createExternalSurveyText} />}
-            description={
-              <FormattedMessage {...messages.createSurveyDescription} />
-            }
-          />
-        }
-      />
-    )}
+          {showSurveys && (
+            <ParticipationMethodRadio
+              onChange={handleParticipationMethodOnChange}
+              currentValue={participation_method}
+              value="survey"
+              name="participationmethod"
+              id={'participationmethod-survey'}
+              label={
+                <LabelHeaderDescription
+                  header={
+                    <FormattedMessage {...messages.createExternalSurveyText} />
+                  }
```

Comment:
Not sure why, but this formatting change was introduced to my PR in a completely unrelated file. Maybe an IDE linting thing?

Reply:
Did you touch this file? Anyway, I believe this change makes sense as it aligns indentations as we usually do.

Reply:
Nope, didn't touch it. Yes, looks fine.